superset/sql_lab.py

```diff
@@ -175,12 +173,11 @@ def get_sql_results(
 
 def execute_sql_statement(sql_statement, query, user_name, session, cursor):
     """Executes a single SQL statement"""
-    query_id = query.id
     database = query.database
     db_engine_spec = database.db_engine_spec
     parsed_query = ParsedQuery(sql_statement)
     sql = parsed_query.stripped()
-    SQL_MAX_ROWS = app.config["SQL_MAX_ROW"]
+    sql_max_rows = app.config["SQL_MAX_ROW"]
```

Comment:
should this be pulled out of the function to the top of the file? Then we keep the same var name but don't need to refetch from the app config each time